extensions/cornerstone/src/services/CornerstoneCacheService/CornerstoneCacheService.ts

```diff
@@ -67,34 +68,42 @@ class CornerstoneCacheService {
   }
 
   public async invalidateViewportData(
-    viewportData: VolumeViewportData,
+    viewportData: StackData | VolumeViewportData,
```

Comment:
why you didn't use the StackViewportData for this? I hate the inconsistensies in the type names here

Reply:
must have picked the wrong type by mistake